prime-dotnet-webapi/Models/PlrProvider.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Prime.Models
+{
+    /// We rely on PLR's database integrity checks rather than enforcing any in our system.
+    [Table("PlrProvider")]
+    public class PlrProvider : BaseAuditable
+    {
+        [Key]
+        public int Id { get; set; }
+
+        public string CollegeId { get; set; }
+
+        public string ProviderType { get; set; }
+
+        public string MspId { get; set; }
+
+        public string NamePrefix { get; set; }
+
+        public string FirstName { get; set; }
+
+        public string SecondName { get; set; }
+
+        public string ThirdName { get; set; }
+
+        public string LastName { get; set; }
+
+        public string Suffix { get; set; }
+
+        public string Gender { get; set; }
+
+        public DateTime DateOfBirth { get; set; }
+
+        public string StatusCode { get; set; }
+
+        public string StatusReasonCode { get; set; }
+
+        public DateTime StatusStartDate { get; set; }
+
+        public DateTime StatusExpiryDate { get; set; }
+
+        public string Expertise { get; set; }
+
+        public string Languages { get; set; }
+
+        public string Address1_Line1 { get; set; }
```

Comment:
I would prefer if we followed standard C# naming conventions and avoided underscores in property names